packages/sdk/snet/sdk/account.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+from snet.snet_cli.utils import normalize_private_key, get_address_from_private, get_contract_object
+
+DEFAULT_GAS = 210000
+
+
+class TransactionError(Exception):
+    """Raised when an Ethereum transaction receipt has a status of 0. Can provide a custom message. Optionally includes receipt"""
+    def __init__(self, message, receipt=None):
+        super().__init__(message)
+        self.message = message
+        self.receipt = receipt
+    def __str__(self):
+        return self.message
+
+
+class Account:
+    def __init__(self, w3, config, mpe_contract):
+        self.config = config
+        self.web3 = w3
+        self.mpe_contract = mpe_contract
+        self.token_contract = get_contract_object(self.web3, "SingularityNetToken.json")
+        private_key = config.get("private_key", None)
+        signer_private_key = config.get("signer_private_key", None)
+        if private_key is not None:
+            self.private_key = normalize_private_key(config["private_key"])
+        if signer_private_key is not None:
+            self.signer_private_key = normalize_private_key(config["signer_private_key"])
+        else:
+            self.signer_private_key = self.private_key
+        self.address = get_address_from_private(self.private_key)
+        self.signer_address = get_address_from_private(self.signer_private_key)
+        self.nonce = 0
+
+
+    def _get_nonce(self):
+        nonce = self.web3.eth.getTransactionCount(self.address)
+        if self.nonce >= nonce:
+            nonce = self.nonce + 1
+        self.nonce = nonce
+        return nonce
+
+
+    def _get_gas_price(self):
+        return self.web3.eth.generateGasPrice()
+
+
+    def _send_signed_transaction(self, contract_fn, *args):
+        transaction = contract_fn(*args).buildTransaction({
+            "chainId": int(self.web3.version.network),
+            "gas": DEFAULT_GAS, 
+            "gasPrice": self._get_gas_price(),
+            "nonce": self._get_nonce()
+        })
+        signed_txn = self.web3.eth.account.signTransaction(transaction, private_key=self.private_key)
+        return self.web3.toHex(self.web3.eth.sendRawTransaction(signed_txn.rawTransaction))
+
+
+    def send_transaction(self, contract_fn, *args):
```

Comment:
Why function for sending transaction is in Account class? In Account class, you mix two entities: 
- class for ETH/MPE account  (analog of MPEAccountCommand from snet-cli)
- class which sends on-chain transactions (analog of ContractCommand from snet-cli)
You probably should reuse contract.py from snet-cli and write better version of ContractCommand.